tasks/linux.sh

```diff
@@ -13,6 +13,7 @@ certname="$PT_certname"
 alt_names="$PT_dns_alt_names"
 custom_attribute="$PT_custom_attribute"
 extension_request="$PT_extension_request"
+jq_missing="jq is required to parse CSR custom attributes or extension requests"
```

Comment:
This seems like an ideal spot to bring in an additional script like one of:

* [TickTick](https://github.com/kristopolous/TickTick) - licensed as MIT *or* Apache 2
* [JSON.sh](https://github.com/dominictarr/JSON.sh) - licensed as MIT *or* Apache 2
* [JSONPath.sh](https://github.com/mclarkson/JSONPath.sh) - licensed as MIT *or* Apache 2
* [JSON.awk](https://github.com/step-/JSON.awk) - licensed as MIT *or* Apache 2

Not sure if you've given this any thought yet @MikaelSmith / @adreyer ?

Reply:
Those would be interesting. We need a way to include multiple files in a task, or a packaging step to merge everything into one file.

If we had multiple files in a task, jq is portable enough we could probably include the binary.